src/lein_dependency_check/core.clj

```diff
@@ -68,8 +72,9 @@
         generator (ReportGenerator. report-name
```

Comment:
The report generator should not be directly used anymore by the plugins. Instead, wrapper methods were moved into the `engine` object ([engine.writeReports()](https://github.com/jeremylong/DependencyCheck/blob/master/dependency-check-core/src/main/java/org/owasp/dependencycheck/Engine.java#L1108))